python/bento_meta/object_map.py

```diff
@@ -474,7 +474,6 @@ def rm_attr_q(self,obj,att,values=None):
           else:
             qry = "MATCH (n:{lbl}){rel}(a) WHERE id(n)={neoid} AND id(a)={aneoid} AND ({cond}) DELETE r RETURN id(n),id(a)".format(
```

Comment:
do we want to include `aneoid` as a param? if `aneoid` is not included, I'm concerned it could it be evaluated as `None` in python and thus `NULL` in Neo4j and thus cause it _not_ to delete relationship?

Using the MAJ1/test-db-bento-meta image, I'm comparing:
```MATCH (n:node) -[r:has_property]->(p:property) WHERE id(n)=5003 and id(p)=5225 DELETE r RETURN id(n),id(p)```
with
```MATCH (n:node) -[r:has_property]->(p:property) WHERE id(n)=5003 and id(p)='' DELETE r RETURN id(n),id(p)```